src/tests/epics/epicSelection.ts

```diff
@@ -183,68 +184,81 @@ export interface Result {
+            inCorrectCohort(userCohorts),
+            excludeSection,
+            excludeTags,
+            hasCountryCode,
+            matchesCountryGroups,
+            // For the super mode pass, we treat all tests as "always ask" so disable this filter
+            ...(isSuperModePass ? [] : [withinMaxViews(targeting.epicViewLog || [])]),
+            withinArticleViewedSettings(targeting.weeklyArticleHistory || []),
+            respectArticleCountOptOut,
+        ];
+    };
+
+    const filterTests = (tests: EpicTest[], filters: Filter[]): EpicTest | undefined => {
+        const test = tests.find(test =>
+            filters.every(filter => {
+                const got = filter.test(test, targeting);
+
+                if (debug[test.name]) {
+                    (debug[test.name] as FilterResults)[filter.id] = got;
+                } else {
+                    debug[test.name] = { [filter.id]: got };
+                }
+
+                if (!got && process.env.LOG_FAILED_TEST_FILTER === 'true') {
+                    console.log(`filter failed: ${test.name}; ${filter.id}`);
+                }
+
+                return got;
+            }),
+        );
+
+        return test;
+    };
+
+    const filterTestsWithSuperModePass = (tests: EpicTest[]): EpicTest | undefined => {
+        return filterTests(tests, getFilters(false)) ?? filterTests(tests, getFilters(true));
+    };
+
+    const filterTestsWithoutSuperModePass = (tests: EpicTest[]): EpicTest | undefined => {
+        return filterTests(tests, getFilters(false));
+    };
 
     const priorityOrdered = ([] as EpicTest[]).concat(
         tests.filter(test => test.highPriority),
         tests.filter(test => !test.highPriority),
     );
 
-    const test = priorityOrdered.find(test =>
-        filters.every(filter => {
-            const got = filter.test(test, targeting);
+    const isSuperMode = targeting.url && isInSuperMode(targeting.url, superModeArticles);
 
-            if (debug[test.name]) {
-                (debug[test.name] as FilterResults)[filter.id] = got;
-            } else {
-                debug[test.name] = { [filter.id]: got };
-            }
-
-            if (!got && process.env.LOG_FAILED_TEST_FILTER === 'true') {
-                console.log(`filter failed: ${test.name}; ${filter.id}`);
-            }
-
-            return got;
-        }),
-    );
+    const test = isSuperMode
+        ? filterTestsWithSuperModePass(priorityOrdered)
+        : filterTestsWithoutSuperModePass(priorityOrdered);
 
     if (test) {
         const variant: EpicVariant = selectVariant(test, targeting.mvtId || 1);
 
-        const shouldThrottleVariant =
-            !!variant.maxViews &&
-            shouldThrottle(targeting.epicViewLog || [], variant.maxViews, test.name);
-
-        if (!shouldThrottleVariant) {
-            return {
-                result: { test, variant },
-                debug: includeDebug ? debug : undefined,
-            };
```

Comment:
It's a little awkard to have variant level throttling + super mode. I think it's best if we just remove this for now as it's unused.